xml/System.IO.Pipelines/ReadResult.xml

```diff
@@ -88,9 +88,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>
-          <see langword="true" /> if the current <see cref="M:System.IO.Pipelines.PipeReader.ReadAsync(System.Threading.CancellationToken)" /> operation was canceled; otherwise, <see langword="false" />.</summary>
-        <value>To be added.</value>
+        <summary>Gets a value that indicates whether current <see cref="M:System.IO.Pipelines.PipeReader.ReadAsync(System.Threading.CancellationToken)" /> operation was canceled.</summary>
+        <value><see langword="true" /> if the current <see cref="M:System.IO.Pipelines.PipeReader.ReadAsync(System.Threading.CancellationToken)" /> operation was canceled; otherwise, <see langword="false" />.</value>
```

Comment:
See https://github.com/dotnet/dotnet-api-docs/pull/3491/files#r346609367

Reply:
```suggestion
        <value><see langword="true" /> if the <see cref="M:System.IO.Pipelines.PipeReader.ReadAsync(System.Threading.CancellationToken)" /> operation that produced this <see cref="T:System.IO.Pipelines.ReadResult" /> was canceled by <see cref="M:System.IO.Pipelines.PipeReader.CancelPendingRead()" />; otherwise, <see langword="false" />.</value>
```